docs/extending.rst

```diff
@@ -270,13 +270,14 @@ Normally, unspecified field names are ignored by the validator. If you would lik
 
         @validates_schema(pass_original=True)
         def check_unknown_fields(self, data, original_data):
-            for key in original_data:
-                if key not in self.fields:
-                    raise ValidationError('Unknown field name {}'.format(key))
+            unknown = set(original_data) - set(self.fields)
+            if unknown:
+                errors = list(map("Unknown field '{}'".format, unknown))
+                raise marshmallow.ValidationError(errors)
```

Comment:
Oh.  I see.  Yes, that makes sense.   `list` is superfluous then, as `set` is already iterable.